collect_app/src/main/java/org/odk/collect/android/preferences/AggregatePreferencesFragment.java

```diff
@@ -176,9 +176,7 @@ public boolean onPreferenceChange(Preference preference, Object newValue) {
                         setupUrlDropdownAdapter();
                     }
                 } else {
-                    Toast.makeText(getActivity().getApplicationContext(),
-                            R.string.url_error, Toast.LENGTH_SHORT)
-                            .show();
+                    ToastUtils.shortDuration(String.valueOf(R.string.url_error));
```

Comment:
An overloaded ToastUtils.makeShort/LongText method that takes the “R” value would save a lot of “String.valueOf”s in the callers.